roles/dosomething.web/tasks/php-thirdparty.yml

```diff
@@ -37,5 +37,7 @@
     dest=/etc/php5/fpm/conf.d/20-newrelic.ini
```

Comment:
NewRelic still uses the package called `php5`, but it supports both php5 and php7.x. https://discuss.newrelic.com/t/repository-for-newrelic-php7/39794/2